conan/cli/commands/version.py

```diff
@@ -12,10 +12,21 @@ def version(conan_api, parser, *args):
     Give information about the Conan client version.
     """
 
-    return {'version': str(conan_version),
+    return {
+            'version': str(conan_version),
+            'conan_script_path': sys.argv[0],
             'python': {
                 'version': platform.python_version().replace('\n', ''),
                 'sys_version': sys.version.replace('\n', ''),
+                'sys_executable': sys.executable,
+                'is_frozen': getattr(sys, 'frozen', False),
+                'architecture': platform.machine(),
+                },
+            'system': {
+                'version': platform.version(),
+                'platform': platform.platform(),
+                'os': platform.system() + ' ' + platform.release(),
```

Comment:
These two can return an empty string. 

If release is empty, then the field would be `"Linux "` which might be tricky for downstream users to properly parse if that happens to them :)

Reply:
Do we prefer to divide it in two? something like this:
```suggestion
                'version': platform.version(),
                'platform': platform.platform(),
                'os_system': platform.system(),
                'os_release':platform.release(),
```

Reply:
Reads better, but not too sold on the names with the prefix. No strong opinions either!

Reply:
Probably just ``system`` and ``release`` is good. 
I'd say better them separately, this is structured information, better separated than us artificially joining them with ``_``